assets/templates/dining_lists/dining_day.html

```diff
@@ -72,10 +72,32 @@ <h4 class="my-0 {% if date.upcoming == date %}font-weight-bold{% endif %}">
             {% include 'dining_lists/snippet_diningslot.html' %}
         {% endfor %}
     </div>
-    {% if can_create_slot %}
-        <a href="{% url 'new_slot' year=date.year month=date.month day=date.day %}"
-           class="col-12 btn btn-block btn-primary">Create and claim a new slot</a>
-    {% endif %}
+
+    <div class="row">
+        <div class="btn-group col-12 d-md-inline-flex d-none">
+            <a class="btn btn-secondary col-3"
+               href="{% url 'credits:transaction_donate' %}?redirect={{ request.path }}">
+                <i class="fas fa-coins"></i> Single use
+            </a>
+            {% if can_create_slot %}
+                <a href="{% url 'new_slot' year=date.year month=date.month day=date.day %}"
+                   class="btn btn-primary">Create and claim a new slot</a>
+            {% endif %}
+        </div>
+
+        <div class="col-12 d-md-none d-inline">
+            {% if can_create_slot %}
+                <a href="{% url 'new_slot' year=date.year month=date.month day=date.day %}"
+                   class="btn btn-primary col-12">Create and claim a new slot</a>
+            {% endif %}
+            <a class="btn btn-secondary col-5 mt-2"
+               href="{% url 'credits:transaction_donate' %}?redirect={{ request.path }}">
+                <i class="fas fa-coins"></i> Single use
+            </a>
+        </div>
```

Comment:
Here you have `col`s inside of `col`s while `col`s should be inside `row`s

Reply:
Some suggestions:
* col-12/row cancel out
* content should be inside cols, e.g. `btn-group` should be on a div inside the col div, not on the same div